src/test/java/com/exasol/adapter/dialects/scalarfunction/ScalarFunctionsAbstractIT.java

```diff
@@ -34,7 +34,7 @@
  */
 @Testcontainers
 @SuppressWarnings("java:S5786") // class is public since it is an abstract test
-@Execution(value = ExecutionMode.SAME_THREAD)
+@Execution(value = ExecutionMode.CONCURRENT) // run with -Djunit.jupiter.execution.parallel.enabled=true
```

Comment:
`// run with -Djunit.jupiter.execution.parallel.enabled=true`
What does the comment mean? run it from where? when?
Should we add a small explanation in the `design.md` file?

Reply:
moved to javadoc